substrate/utils/frame/remote-externalities/src/lib.rs

```diff
@@ -1264,6 +1264,16 @@ mod tests {
 mod remote_tests {
 	use super::test_prelude::*;
 	use std::os::unix::fs::MetadataExt;
+	use std::env;
+
+	static TRANSPORT_URL: once_cell::sync::Lazy<String> = once_cell::sync::Lazy::new(|| {
+        env::var("TEST_WS").unwrap_or_else(|_| DEFAULT_HTTP_ENDPOINT.to_string())
+    });
+
+	#[test]
+	fn print_transport_url() {
+		println!("TRANSPORT_URL: {:?}", once_cell::sync::Lazy::<std::string::String>::force(&TRANSPORT_URL));
+	}
```

Comment:
why's this needed?

Reply:
I found that when running the tests, I would look at the output but not know which `TRANSPORT_URL` it was using.

If you think it is not needed, I can remove.

Reply:
I wonder if it's possible to log in the closure where the static var is defined?

Reply:
That was what I tried to do first but with no avail :(